tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,24 @@
+#replace with 8,16,32 nodes, test with 2 compute nodes, injected into pcluster config reader
```

Comment:
When a comment is there to take some future action, prepend a TODO or a TOFIX.
futhermore it's better to place the comment close to the line where you will be supposed to take action. In this case just above the line where the number of nodes are defined.